### PR TITLE
Ditch glide

### PR DESCRIPTION
**Pull request by melekes**

### CODE REVIEW

**Comment by melekes** [comment]
```
Error: unknown command "kvstore" for "abci-cli"
Run 'abci-cli --help' for usage.
```

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -34,15 +34,26 @@ Please don't make Pull Requests to `master`.
...
 
-Run `bash scripts/glide/status.sh` to get a list of vendored dependencies that may not be up-to-date.
+That said, the master branch of every Tendermint repository should just build
+with `go get`, which means they should be kept up-to-date with their
+dependencies so we can get away with telling people they can just `go get` our
+software.
+
+Since some dependencies are not under our control, a third party may break our
+build, in which case we can fall back on `dep ensure` (or `make
+get_vendor_deps`). Even for dependencies under our control, dep helps us keeps
```
Comment (xla) [inline CONTRIBUTING.md]: `keeps` -> `to keep`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -53,11 +64,14 @@ vagrant ssh
 make test
 ```
 
-
 ## Testing
 
 All repos should be hooked up to circle.
```
Comment (xla) [inline CONTRIBUTING.md]: `[CircleCI](https://circleci.com/)`

File: scripts/dep_utils/checkout.sh [hunk 3]
```diff
@@ -0,0 +1,32 @@
+#! /bin/bash
+
+set +u
+if [[ "$DEP" == "" ]]; then
+		DEP=$GOPATH/src/github.com/tendermint/tendermint/Gopkg.lock
```
Comment (xla) [inline scripts/dep_utils/checkout.sh]: Indentation not consistent.

File: scripts/dep_utils/checkout.sh [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+fi
+set -u
+
+set -u
```
Comment (xla) [inline scripts/dep_utils/checkout.sh]: Duplicate of line 7?

File: scripts/dep_utils/parse.sh [hunk 5]
```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
+
+set +u
+if [[ "$DEP" == "" ]]; then
+		DEP=$GOPATH/src/github.com/tendermint/tendermint/Gopkg.lock
+fi
+set -u
+
+set -euo pipefail
```
Comment (xla) [inline scripts/dep_utils/parse.sh]: Can be merged with line 7?

**Pull request merged**